xbmc/epg/EpgInfoTag.cpp

```diff
@@ -749,3 +756,13 @@ void CEpgInfoTag::SetEpg(CEpg *epg)
 {
   m_epg = epg;
 }
+
+bool CEpgInfoTag::IsSeries(void) const
+{
+  return m_iFlags & EPG_TAG_FLAG_IS_SERIES > 0;
+}
```

Comment:
Sorry just to clarify my (lack of) understanding... do you mean to change it to the following, and move it to header file instead of cpp file?

```
inline bool IsSeries() { return m_iFlags & EPG_TAG_FLAG_IS_SERIES > 0; }
```